bootstraped-multi-test-results-report/src/main/java/com/github/bogdanlivadariu/jenkins/reporting/cucumber/CucumberTestReportPublisher.java

```diff
@@ -117,7 +117,7 @@ public boolean perform(AbstractBuild build, Launcher launcher, BuildListener lis
             new FilePath(targetBuildJsonDirectory));
 
         // generate the reports from the targetBuildDirectory
-        Result result = Result.NOT_BUILT;
```

Comment:
leave the `Result.NOT_BUILT` as being the default value, so that jenkins will set the build status properly